ts/types/services/ServicePreference.ts

```diff
@@ -0,0 +1,15 @@
+import { IUnitTag } from "italia-ts-commons/lib/units";
+
+/**
+ * The unique ID of a Service
+ */
+export type ServiceID = string & IUnitTag<"ServiceID">;
+
+/**
+ * The contact preference related to a service
+ */
+export type ServicePreference = {
```

Comment:
We already have a similar definition
https://github.com/pagopa/io-app/blob/ac6e9ef811711a60d8f6cc5733051bb5c2b07788/ts/utils/profile.ts#L101
Could we use only one of them?

Reply:
didn't know about this! I'll use it here